howdoi/howdoi.py

```diff
@@ -432,6 +434,57 @@ def _get_cache_key(args):
     return str(args) + __version__
 
 
+def print_stash(sl = []):
+    if len(sl) == 0:
+        stash_list = ['\nSTASH LIST:']
+        commands = keep_utils.read_commands()
+
+        if commands is None or len(commands.items()) == 0:
+            print('No commands found in stash. Add a command with "howdoi -save <query>".')
+            return
+
+        for cmd, fields in commands.items():
+            stash_list.append(
+                '\033[4m\033[1m$ ' + fields['alias'] + '\033[0m\n\n' 
```

Comment:
We should probably add some comments explaining what this is -- if someone sees this they'll definitely question what it means.